ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoice.java

```diff
@@ -383,6 +389,7 @@ private SafeFuture<BlockImportResult> onBlock(
       final SignedBeaconBlock block,
       final Optional<BeaconState> blockSlotState,
       final Optional<BlockImportPerformance> blockImportPerformance,
+      final Optional<SafeFuture<BlockImportResult>> consensusValidation,
```

Comment:
Here and on all the way of this var I think we should better clarify it's purpose. Maybe something like `consensusValidationResultCallback`?